docs/source/dev_guide.rst

```diff
@@ -6,13 +6,9 @@ Developer's Guide to Memex Explorer
 Setting up Memex Explorer
 *************************
 
-To setup your machine, you will need Anaconda or Miniconda
-installed. Miniconda is a minimal Anaconda installation that
-bootstraps conda and Python on any operating system. Install `Anaconda
-<http://continuum.io/downloads>`_ or `Miniconda
-<http://conda.pydata.org/miniconda.html>`_ from their respective sites.
+To setup your machine, you will need Anaconda or Miniconda installed. Miniconda is a minimal Anaconda installation that bootstraps conda and Python on any operating system. Install `Anaconda <http://continuum.io/downloads>`_ or `Miniconda <http://conda.pydata.org/miniconda.html>`_ from their respective sites.
```

Comment:
I'll defer to your expertise, I think this is just a stale word bug that we inherited in the docs.